.changeset/dialog-backdrop-type-re.md

```diff
@@ -0,0 +1,5 @@
+---
+"ariakit": minor
+---
+
+Updated the `backdrop` prop type on `Dialog` again to remove the original type which is a subset of the new type.
```

Comment:
Can you delete this file and edit the original changeset to include this PR reference? The change hasn't been released yet, so we don't need two changelog entries. :)

Reply:
Commit pushed! New to this Changeset stuff, am I doing it right? 👀

Reply:
Yeah, that's right!